operator/controllers/manifest_controller.go

```diff
@@ -174,7 +174,16 @@ func (r *ManifestReconciler) sendJobToInstallChannel(ctx context.Context, logger
 		Client: r.Client, Config: r.RestConfig,
 	}, r.ReconcileFlagConfig)
 	if err != nil {
-		return err
+		logger.Error(err, fmt.Sprintf("cannot prepare install information for %s resource %s",
+			v1alpha1.ManifestKind, namespacedName))
+		if mode == manifest.DeletionMode {
+			// since installation path for manifest could not be determined
```

Comment:
This comment is hard to understand, what does this case cover?

Reply:
I improved the comment here, let me know if it's more clear